### PR TITLE
Add small dots under title and fix bugs

### PR DESCRIPTION
**Pull request by daming-lu**
screen shot 2018-06-22 at 3 39 38 pm

### CODE REVIEW

File: portal/portal/templates/index.html [hunk 1]
```diff
@@ -161,6 +161,14 @@ <h1 class="col-lg-12">Get Started</h1>
         <div class="container">
             <div class="row models-slogan">
                 <h2 class="col-lg-12"><span>{% trans "Build powerful applications with PaddlePaddle"%}</span></h2>
+                <div class="six-dots col">
```
Comment (jetfuel) [inline portal/portal/templates/index.html]: I remember @varunarora would like to remove the dots, we probably need to update this part later.

Comment (daming-lu) [reply portal/portal/templates/index.html]: Sure. I can remove them if his Highest does not like it 🛐

Comment (varunarora) [reply portal/portal/templates/index.html]: oh no i wanna keep them!

Comment (varunarora) [reply portal/portal/templates/index.html]: I increased them to make it look not like a carousel

Comment (daming-lu) [reply portal/portal/templates/index.html]: @varunarora : right now the dot are 10 x 10 px. We can tune them in staging and I can update the code 👑

**Pull request merged**